modules/nf-core/bowtie/align/tests/main.nf.test.snap

```diff
@@ -0,0 +1,192 @@
+{
+    "sarscov2 - single_end": {
+        "content": [
+            [
+                "versions.yml:md5,96e36b0b99c80da0be8239d03db30ecc"
+            ],
+            [
+                "7bdcfc6f54ae6e8f4570395cc85db9a3"
+            ],
+            [
```

Comment:
maybe stupid nf-test question: why do the content entries of the stub tests get keys, but not the non-stub?

Reply:
It means the test isn't just the md5 on the whole channel

Reply:
interesting thanks!